docs/src/development/local/ddev.md

```diff
@@ -0,0 +1,143 @@
+
+Test the configuration by running the following command:
+
+```bash
+ddev start
+```
+
+Your project should start running, though without any of the data from your Platform.sh environment.
+
+## 3. Add an API token
+
+To connect DDEV with your Platform.sh account, use a Platform.sh API token.
+
+1. [Create an API token](./../cli/api-tokens.md#get-a-token) in the console.
+2. Add the token to your global DDEV configuration:
+
+   ```yaml {location="~/.ddev/global_config.yaml"}
+   web_environment:
+       - PLATFORMSH_CLI_TOKEN=<API_TOKEN>
+   ```
+
+   Note that this isn't the configuration file in your project.
+3. Get DDEV to recognize the token by restarting DDEV:
+
+   ```bash
+   ddev restart
+   ```
+
+Now you can run `ddev exec platform <command>` from your computer without needing to install the Platform.sh CLI.
+
+## 4. Connect DDEV to your project
+
+To get DDEV connected to your project, you need to tell it which project to use:
+
+1. In your project's `.ddev/providers` directory, make a copy of `platform.yaml.example` and name it `platform.yaml`.
+
+   ```bash
+   cp .ddev/providers/platform.yaml.example .ddev/providers/platform.yaml
+   ```
+
+2. Update `environment_variables` with your project ID and environment name.
+   If you don't know your project ID, run `ddev exec platform`.
+
+## 5. Get your project data
+
+To get your environment data (files, database), run the following command:
+
+```bash
+ddev pull platform
+```
+
+If you have configured your files in a mount other than the default,
+adjust the commend in `.ddev/providers/platform.yaml` for the correct location.
+
+To skip pulling files, add `--skip-files` to the command.
+To skip pulling a database, add `--skip-db` to the command.
+
+## 6. (Optional) Install dependencies
+
+If your project has dependencies, you can install them with Composer.
+Using the DDEV version of Composer means you don't need to install Composer on your machine.
+You can also use different Composer versions in different projects based on their DDEV configurations.
+
+Run the following command:
+
+```bash
+ddev composer install
+```
+
+If you have any other commands in your [build hook](../../create-apps/hooks/_index.md),
+you need to run those explicitly.
+You can also add them as [DDEV hooks in your configuration](https://ddev.readthedocs.io/en/stable/users/extending-commands/).
+
+## 7. Run your project
+
+Now your project is ready to run:
+
+```bash
+ddev start
+```
+
+The command returns the project URL `https://<PROJECT_NAME>.ddev.site/`
+as well as a specific port on `https://127.0.0.1`.
+Open one of the URLs to see your project running.
+
+## What's next
+
+You've got a project running on a local web server.
+Now you can add customizations.
+For more ideas and options, see the [DDEV documentation](https://ddev.readthedocs.io/en/stable/).
+
+### Add environment variables
+
+If your project requires environment variables for its hooks or runtime,
+add them to the DDEV environment:
```

Comment:
:nail_care: ▶️
As opposed to the global DDEV environment setting in [3. Add an API token](https://pr-2325-r3lvrhi-652soceglkw4u.eu-3.platformsh.site/development/local/ddev.html#3-add-an-api-token).
```suggestion
add them to the local DDEV environment:
```

Reply:
I can see how it would help to clarify, but given that "local" is true for both (for everything DDEV), I'll try to find a different qualifier. :thinking: